index.html

```diff
@@ -4,430 +4,336 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ben Omayio</title>
+    <title>Ben Omayio Portfolio</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
+
   <body>
```

Comment:
- [ Optional ] The is scroll effect while the mobile menu is open. what do you think stopping scroll effect better look and feel? Take a look at this snapshot taken after a bit of down scroll.
![mobile-body-scroll-on-open-menu](https://user-images.githubusercontent.com/44057863/167739601-6a90ad8c-9002-4f06-82bd-d0dc687eedc8.png)